packages/browser-wallet/src/background/identity-issuance.ts

```diff
@@ -60,20 +61,46 @@ export function addIdpListeners() {
         }
     });
 
-    const handleIdpRequest = (redirectUrl: string, tabId: number) => {
-        chrome.tabs.remove(tabId);
-        sessionIdpTab.remove();
+    const handleIdpRequest = (redirectUrl: string, tabId?: number) => {
+        if (tabId !== undefined) {
+            chrome.tabs.remove(tabId);
+            sessionIdpTab.remove();
+        }
```

Comment:
Do we want this to run for error requests with tabID -1?

Reply:
As far as I understand the "protocol", the error request only happens before we set it, so the tabId will be undefined. But i've changed the condition to be more obvious.